template/footer.mustache

```diff
@@ -1,5 +1,80 @@
-Powered by
-<a href='https://github.com/OneHourGameJam/OneHourGameJam'>One Hour Game Jam</a> version {{SOFTWARE_VERSION}} -
+<!-- New footer design -->
+<footer class="footer-distributed">
+
+    <div class="footer-left">
+
+        <img src="template/images/logos/logo_onehourgamejam.png" alt="Logo" class="mainLogo">
+        <p class="footer-links">
+            <a href='?{{{forms.get.page}}}={{{forms.pages.main}}}' class="link-1">Home</a>
+            
+            <a href='?{{{forms.get.page}}}={{{forms.pages.rules}}}'>How it works</a>
+                
+
+        </p>
+
+        <p class="footer-company-name">OneHourJam 2021</p>
+    </div>
+
+    <div class="footer-center">
+        <br>
+        <br>
+        Powered By - <br>
+        <a href='https://github.com/OneHourGameJam/OneHourGameJam'>One Hour Game Jam</a> version {{SOFTWARE_VERSION}} -
 <a href='https://github.com/OneHourGameJam/OneHourGameJam/issues' target='_BLANK'>Report a bug / suggest a feature</a>
-<br />
-{{{CONFIG.VALUES.SITE_FOOTER}}}
\ No newline at end of file
+
+    </div>
+
+    <div class="footer-right">
+
+        <p class="footer-company-about">
+            <span>About OneHourGameJam</span>
+            {{{CONFIG.VALUES.SITE_FOOTER}}}
+        </p>
+
+        <div class="footer-icons" id="mainmenu-social">
+            {{#CONFIG.VALUES.TWITTER_ACCOUNT}}
+                <div class="footer-icon-right">
+                    <a href="https://twitter.com/{{CONFIG.VALUES.TWITTER_ACCOUNT}}" target='_blank'>
+                        <div class='mediaButton twitterButton'><img src='{{{page.template_path}}}images/twitter.png' class="menuButton" /> Follow</div>
+                    </a>
+                </div>
+                {{/CONFIG.VALUES.TWITTER_ACCOUNT}}
+                {{#CONFIG.VALUES.DISCORD_INVITE_URL}}
+                <div class="footer-icon-right">
+                    <a href="{{{CONFIG.VALUES.DISCORD_INVITE_URL}}}" target='_blank'>
+                        <div class='mediaButton discordButton'><img src='{{{page.template_path}}}images/discord.png' class="menuButton" /> Discord</div>
+                    </a>
+                </div>
+                {{/CONFIG.VALUES.DISCORD_INVITE_URL}}
+                {{#CONFIG.VALUES.TWITCH_ACCOUNT}}
```

Comment:
Indentation - should be in-line with `{{#CONFIG.VALUES.TWITTER_ACCOUNT}}` (Not required to fix, informational)